server/controllers/contact_list_controller.js

```diff
@@ -5,11 +5,52 @@ const path = require('path');
 const Settings = require('../../settings');
 const optIn = 'opt-in';
 
+function prepareNotificationEmail(reqBody) {
```

Comment:
Duplicative function name.  Looks like the other one is the correct one.  Deleting this version of the function def